lib/ansible/plugins/terminal/vyos.py

```diff
@@ -38,9 +38,12 @@ class TerminalModule(TerminalBase):
         re.compile(r"\n\s+Set failed"),
     ]
 
+    terminal_length = os.getenv('ANSIBLE_VYOS_TERMINAL_LENGTH', 10000)
```

Comment:
Thinking about this a bit more, should this be moved to `lib/ansible/constants.py`